x/slashing/handler_test.go

```diff
@@ -53,19 +53,56 @@ func TestCannotUnjailUnlessMeetMinSelfDelegation(t *testing.T) {
 
 	undelegateMsg := staking.NewMsgUndelegate(sdk.AccAddress(addr), addr, sdk.OneDec())
 	got = staking.NewHandler(sk)(ctx, undelegateMsg)
-
+	require.True(t, got.IsOK())
 	require.True(t, sk.Validator(ctx, addr).GetJailed())
 
-	// assert non-jailed validator can't be unjailed
+	// assert jailed validator can't be unjailed due to min self-delegation
 	got = slh(ctx, NewMsgUnjail(addr))
 	require.False(t, got.IsOK(), "allowed unjail of validator with less than MinSelfDelegation")
 	require.EqualValues(t, CodeValidatorNotJailed, got.Code)
 	require.EqualValues(t, DefaultCodespace, got.Codespace)
 }
 
+func TestUnjailNoTransferValidator(t *testing.T) {
+	// initial setup
+	ctx, bk, sk, _, keeper := createTestInput(t, DefaultParams())
+	keeper.bk = mockBankKeeper{false}
+
+	slh := NewHandler(keeper)
+	amtInt := int64(100)
+	delAddr := sdk.AccAddress(addrs[0])
```

Comment:
clarity could be improved (var names or comments), I'm assuming that `delAddr` is the validator "sponsor" so to speak - could be good to include this terminology in the variable name or comment or something